scripts/imageusagechecker.py

```diff
@@ -0,0 +1,74 @@
+import os
+import sys
+import hashlib
+import argparse
+import importlib
+import re
+
+IMAGE_FORMATS = ['.png', '.jpg', '.svg']
+
+
+def clean_module_path(path):
+    if path.endswith(".py"):
+        path = path[:-len(".py")]
+
+    return path.replace("/", ".").replace("\\", ".")
+
+def build_hashes(dir):
+    located_files = {}
+
+    for root, subFolders, files in os.walk(dir):
+        for file in files:
+            filename, file_extension = os.path.splitext(file)
+
+            if file_extension in IMAGE_FORMATS:
+                filepath = os.path.join(root, file)
+                with open(filepath, 'rb') as afile:
+                    hasher = hashlib.md5()
+
+                    hasher.update(afile.read())
+                    located_files[filepath] = hasher.hexdigest()
```

Comment:
```suggestion
                    located_files[filepath] = hashlib.md5(afile.read()).hexdigest()
```
Only optional, both should work

Reply:
I'd rather keep the readability.